### ISSUE TITLE
Add support for tss-react/mui

### ISSUE THREAD

**Issue by felixfon** [stylelint/postcss-css-in-js#287]
### What is the problem you're trying to solve?

Consider the following code snippet:
```JSX
import React from 'react';
import { makeStyles } from 'tss-react/mui';
import Button from '@material-ui/core/Button';

const useStyles = makeStyles({
  name: {
    MyComponent,
  },
})({
  root: {
    background: 'linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)',
    border: 0,
    borderRadius: 3,
    boxShadow: '0 3px 5px 2px rgba(255, 105, 135, .3)',
    color: 'white',
    height: 48,
    padding: '0 30px',
  },
});

export default function MyComponent() {
  const classes = useStyles();
  return <Button className={classes.root}>MyComponent</Button>;
}
```

When passed into postcss-css-in-js through styelint, the styles in the function returned from makeStyles function are ignored.

### What solution would you like to see?

I expect a warning for the styles that use `tss-react`, but not for the argument for makeStyles.
Because the argument for makeStyles is used to name the stylesheets, it's not a style.

FYI: https://github.com/stylelint/postcss-css-in-js/compare/master...felixfon:postcss-css-in-js:tss-react
